site/content/entry/yuzu-tcr/index.md

```diff
@@ -0,0 +1,168 @@
+  >}}
+
+## Current limitations and future progress
+
+Thanks to these changes, hardware lacking the `VK_EXT_robustness2` extension will not produce the optimal experience. In Windows, this includes AMD graphics cards older than Vega (Polaris and older series) and all Intel iGPUs to date. You can check the current support [here](http://vulkan.gpuinfo.org/listdevicescoverage.php?extension=VK_EXT_robustness2&platform=windows). Games requesting this extension on unsupported hardware may behave randomly or crash in rare instances. A fallback code path is being worked on. Make sure to be up to date with your drivers, as the GPU vendor may be able to add support in the future if the hardware allows it.
+
+Originally, `Bindless Texture` support was expected to be added, but several difficulties emerged during development. One of the problems is the lack of native hardware support for ASTC texture decoding. If we used uncompressed textures, GPUs with less than 8GB of VRAM would not be able to fit the game assets, and if we recompressed in another texture format to avoid this problem, image quality would degrade. True bindless texture support can be considered again in the future.
+
+`Depth Stencil Blits` are not implemented on Vulkan for devices that don't offer native support (any AMD and Intel GPU).
+
+Another complication that emerged during development is related to memory management. The idea was to release the Texture Cache Rewrite with what the team calls the `Texture Reaper`, a way to remove from VRAM textures that has not been used after some time. While this has been almost working in OpenGL during testing, managing to run Luigi's Mansion 3 in under 300MB of VRAM, Vulkan on the other hand received no benefit.
+
+Vulkan faces two main problems, it fragments the memory, and textures need to be mapped to contiguous video memory. There’s no tolerance for fragmentation, so freeing blocks will not help at all if the next texture doesn’t fit in the new empty space. This will require the development of a VRAM defragmentation routine, work that can take quite some time. So we can say that today marks the day `Project Texture Reaper` starts.
+
+A feature that will be added shortly later will be `Accelerated Texture Decoding`, handling any texture format via `Compute Shaders`, even formats the GPU doesn’t support natively. [epicboy](https://github.com/ameerj) is working on the ASTC compute decoder.
+
+The next project [Rodrigo](https://github.com/ReinUsesLisp) is working on is the `Buffer Cache Rewrite`. This work promises to solve more rendering issues (for example font rendering problems) and seriously improve performance, especially on memory bandwidth starved hardware like integrated GPUs.
+
+That's all folks! Please report to us any new bugs or problems you face with this new feature in the [Forums](https://community.citra-emu.org/c/yuzu-support/) or in our [Discord server!](https://discord.gg/u77vRWY)
```

Comment:
Change `That's all folks` to `That's the overview for the Texture Cache Rewrite, folks!`

Reply:
You are making me change my merrie melodies ending?!